src/dnx.common/tpa.cpp

```diff
@@ -23,6 +23,7 @@ const std::vector<const dnx::char_t*> CreateTpaBase(bool native_images)
             _X("System.Collections.ni.dll"),
             _X("System.Collections.Concurrent.ni.dll"),
             _X("System.ComponentModel.ni.dll"),
+            _X("System.Console.ni.dll"),
```

Comment:
Don't check this in. I removed it in another commit. Something strange going on or we have different CoreCLR builds... Or something else is going on..